packages/babel-register/src/cache.js

```diff
@@ -43,6 +43,9 @@ export function save() {
     fs.writeFileSync(FILENAME, serialised);
   } catch (e) {
     switch (e.code) {
+      // workaround https://github.com/nodejs/node/issues/31481
+      // todo: remove the ENOENT error check when we drop node.js 13 support
+      case "ENOENT":
```

Comment:
Note that it's okay to place `ENOENT` here because `write(2)` will never throws `ENOENT`. http://man7.org/linux/man-pages/man2/write.2.html

So `ENOENT` must be thrown from `fs.mkdirSync(#, { recursive: true })`.